proto/evm/v1beta1/types.proto

```diff
@@ -64,7 +65,8 @@ message ERC20TokenMetadata {
       [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Hash" ];
   bytes min_amount = 6 [
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable) = false
+    (gogoproto.nullable) = false,
+    deprecated = true
```

Comment:
It is only used in the migration not in the code anymore. If we removed it, the migration will be messy because we need to write a legacy function to unmarshal the legacy proto type. (check the migration pr, it is pretty neat with the deprecated field) 
We can remove in the future upgrade if you do not have strong opinion to remove it now.